client/my-sites/guided-transfer/issues-notices.jsx

```diff
@@ -38,10 +35,7 @@ class IssuesNotices extends Component {
 				{ customFontIssue && ! customFontIssue.prevents_transfer && (
 					<Notice status="is-warning" showDismiss={ false }>
 						{ translate(
-							`Your site uses a custom font that can't be
-						transferred. Continuing will automatically activate the
-						default font, or you can
-						{{a}}choose a free theme{{/a}}.`,
+							"Your site uses a custom font that can't be transferred. Continuing will automatically activate the default font, or you can {{a}}choose a free theme{{/a}}.",
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **5** times:
`translate( 'Your site uses a custom font that can't be
						transferred. Continuing will automatically activate the
						default font, or you can
						{{a}}choose a free theme{{/a}}.' )` *ES Score: 8*